leetcodePython/SlidingWindow/slidingwindow_1838.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+
+
+class Solution:
+    def maxFrequency(self, nums: List[int], k: int) -> int:
+        nums.sort()
+        current_sum = 0
+        left = 0
+        for right in range(len(nums)):
+            freq_num = nums[right]
+            current_sum += freq_num
+            freq_window_sum = freq_num * (right - left + 1)
```

Comment:
* we can only increment.
* then current max number will be the frequency number

`expect_window_sum = max_num * (right - left + 1)`

`expect_window_sum - current_sum` should have no more than `k` increment. Or we have to shrink window from `left`